gapic/src/main/com/google/gapid/perfetto/views/RootPanel.java

```diff
@@ -359,6 +360,19 @@ protected void finishSelection(int mods) {
 
   @Override
   public Hover onMouseMove(Fonts.TextMeasurer m, double x, double y, int mods) {
+    if (mouseMode == MouseMode.Flagging) {
+      return new Hover() {
```

Comment:
It would be great if you could keep this onMouseMove method concise, you could return this Hover object through a private method.

Reply:
Done